lib/taskana-core/src/main/java/pro/taskana/common/internal/OracleSqlSessionFactory.java

```diff
@@ -0,0 +1,89 @@
+package pro.taskana.common.internal;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+import org.apache.ibatis.session.Configuration;
+import org.apache.ibatis.session.ExecutorType;
+import org.apache.ibatis.session.SqlSession;
+import org.apache.ibatis.session.TransactionIsolationLevel;
+import org.apache.ibatis.session.defaults.DefaultSqlSessionFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class OracleSqlSessionFactory extends DefaultSqlSessionFactory {
+
```

Comment:
should we comment what this is for (because the other databases don't have this)?